docker/clang/Dockerfile-6.0

```diff
@@ -31,6 +31,9 @@ RUN apt-get update && apt-get install -y \
 && curl -Lo /usr/local/bin/git-clang-format https://llvm.org/svn/llvm-project/cfe/trunk/tools/clang-format/git-clang-format \
 && chmod +x /usr/local/bin/git-clang-format
 
+RUN git clone https://github.com/ccache/ccache -b v3.4.3 && \
+    cd ccache && bash autogen.sh && ./configure && make -j 2 && mv ccache /usr/local/bin
```

Comment:
Ugh. Please download the tarball instead of cloning the entire history, do a proper install and clean up after yourself.  Like so:
```
RUN cd /tmp \
 && curl https://www.samba.org/ftp/ccache/ccache-3.4.3.tar.xz | tar xJ \
 && cd ccache-3.4.3 \
 && ./configure --prefix=/usr/local \
 && make && make install \
 && cd \
 && rm -r /tmp/ccache-3.4.3
```
Also, we don't need `autoconf` in that case.

Reply:
I tried but ```make install``` has a weird dependency on asciidoc.